.buildbot.sh

```diff
@@ -39,10 +39,18 @@ cd ..
 rustup override set ykrustc-stage1
 
 # Test both workspaces using the compiler we just built.
+# We do this for both debug and release mode.
 export RUSTFLAGS="-C tracer=${CI_TRACER_KIND} -D warnings"
-cargo xtask test
-cargo xtask bench
-cargo xtask clean
+for mode in "" "--release"; do
+    cargo xtask test ${mode}
+    # check extra args are handled right.
+    cargo xtask test guard ${mode} -- --nocapture
+    cargo xtask clean ${mode}
+
+    # Also test the build without xtask, as that's what consumers will do.
```

Comment:
I thought we were implicitly banning people from not using `xtask`?

Reply:
We've [discussed this recently](https://github.com/softdevteam/yk/pull/290#pullrequestreview-612416859).

You can't ban the use of raw cargo, as that's what crates consuming us will run. There's no way to have consumers run xtask.

Reply:
Point taken.